src/frontend/components/Chat/SharedChatComponents/ChatInputBar/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { SendButtonSVG } from 'components/SVGIcons/SendButtonSVG';
+import { Box, Button, TextInput } from 'grommet';
+import React, { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { converse } from '../../../../utils/window';
+
+const messages = defineMessages({
+  inputBarPlaceholder: {
+    defaultMessage: 'Message...',
+    description:
+      'The input bar to write, edit and send messages to the chat conversation.',
+    id: 'components.InputBar.inputBarPlaceholder',
+  },
+});
+
+const SendButtonStyled = styled(Button)`
+  width: 40px;
+  height: 40px;
+  border: 1px solid;
+  border-color: #035ccd;
+  border-radius: 6px;
+  margin: 5px;
+`;
+
+const TextInputStyled = styled(TextInput)`
+  font-weight: normal;
+  color: #035ccd;
```

Comment:
you should use a color defined in the theme and in this case (because this is a grommet component) check if you can or should style textInput in the global grommet theme

Reply:
I just realized i was using wrong the theme.ts, thanks !
I changed groomet theme and set a color for input text in the theme, as well as I defined a new color in the colorGeneric object.
Theses two colors i defined in the theme.ts are a little different from what i used before, because I checked the mockup another time, and I didn't have the right one (or maybe it changed since the last time)